src/traces/choropleth/plot.js

```diff
@@ -79,54 +43,20 @@ plotChoropleth.plot = function(geo, calcData, geoLayout) {
 
     gChoroplethTraces.each(function(calcTrace) {
         var trace = calcTrace[0].trace,
-            cdi = plotChoropleth.calcGeoJSON(trace, geo.topojson),
-            cleanHoverLabelsFunc = makeCleanHoverLabelsFunc(geo, trace),
-            eventDataFunc = makeEventDataFunc(trace);
-
-        // keep ref to event data in this scope for plotly_unhover
-        var eventData = null;
-
-        function handleMouseOver(pt, ptIndex) {
-            if(!geo.showHover) return;
-
-            var xy = geo.projection(pt.properties.ct);
-            cleanHoverLabelsFunc(pt);
-
-            Fx.loneHover({
-                x: xy[0],
-                y: xy[1],
-                name: pt.nameLabel,
-                text: pt.textLabel
-            }, {
-                container: geo.hoverContainer.node()
-            });
-
-            eventData = eventDataFunc(pt, ptIndex);
+            cdi = calcGeoJSON(trace, geo.topojson);
 
-            geo.graphDiv.emit('plotly_hover', eventData);
-        }
-
-        function handleClick(pt, ptIndex) {
-            geo.graphDiv.emit('plotly_click', eventDataFunc(pt, ptIndex));
-        }
-
-        var paths = d3.select(this).selectAll('path.choroplethlocation')
-                .data(cdi);
+        var paths = d3.select(this)
+            .selectAll('path.choroplethlocation')
+            .data(cdi);
 
         paths.enter().append('path')
             .classed('choroplethlocation', true)
-            .on('mouseover', handleMouseOver)
-            .on('click', handleClick)
-            .on('mouseout', function() {
-                Fx.loneUnhover(geo.hoverContainer);
-
-                geo.graphDiv.emit('plotly_unhover', eventData);
-            })
-            .on('mousedown', function() {
-                // to simulate the 'zoomon' event
-                Fx.loneUnhover(geo.hoverContainer);
+            .on('mouseover', function(pt) {
+                geo.choroplethHoverPt = pt;
```

Comment:
`pt` here is essentially the calcdata item corresponding to the feature under the mouse cursor. `geo.choroplethHoverPt` is then used in the `hoverPoints` method. 

Alternatively, we could have looped over all the features and use something like [`turf-inside`](https://github.com/Turfjs/turf-inside) which is more similar to the `hoveron: 'fills'` algorithm, but I thought this here sufficed.